textrepo-app/src/main/java/nl/knaw/huc/service/task/importer/ImportFileTaskBuilder.java

```diff
@@ -12,7 +14,7 @@
 
   ImportFileTaskBuilder forFilename(String name);
 
-  ImportFileTaskBuilder withContents(byte[] contents);
+  ImportFileTaskBuilder withInputStream(InputStream inputStream);
```

Comment:
volgens mij can methodenaam `withContents` blijven, aangezien inputstream nog steeds de contents van een file version bevat?

Reply:
Done in https://github.com/knaw-huc/textrepo/commit/36346f4cbc5a1b4e46156a5f58d01f97e2aba189